src/Components/__stories__/RelayRouter.story.tsx

```diff
@@ -0,0 +1,61 @@
+import { storiesOf } from "@storybook/react"
+import React, { Component } from "react"
+import { buildClientApp } from "../../Router"
+import { graphql } from "react-relay"
+
+class Container extends Component {
+  state = {
+    ClientApp: null,
+  }
+
+  async componentDidMount() {
+    const routes = [
+      {
+        Component: ({ artist }) => {
+          return (
+            <div>
+              <h1>Example Relay Router App</h1>
+              <h3>{artist.name}</h3>
+              <p>{artist.bio}</p>
+            </div>
+          )
+        },
+        path: "*",
+        query: graphql`
+          query RelayRouterQuery($artistID: String!) {
+            artist(id: $artistID) {
+              name
+              bio
+            }
+          }
+        `,
+        prepareVariables: params => ({
+          artistID: "andy-warhol",
+        }),
+
+        // TODO: Figure out how to get around Storybooks + iFrame. For now only
+        // a single-route example is possible since cant push into top-level history.
+        // children: []
```

Comment:
@alloy - This is a limitation I just discovered in Storybooks -- we can't push to history due to it being a sub-window iframe. Regardless, I added an example app with one route to show how a routing app can be built out in Reaction (meaning, render the page with the top level query and then go from there down the tree).

Reply:
Actually -- kinda scratch that. Farce supports in memory routing updates so i've updated `buildClientApp` to support different history types and updated the example